### ISSUE TITLE
Improve color picking

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/nmr-displayer#736]
The wheel on each spectrum should be replaced with the contextual menu. In this contextual menu there will be currently only one option : delete spectrum.

The current line should be replaced by a square
![image](https://user-images.githubusercontent.com/1484241/101806000-b7437180-3b13-11eb-8429-626343955aca.png)
When clicking on the square you will have the color picker.

**Comment by hamed-musallam** [comment]
@lpatiny 
I agree with you if the spectrum is 1d because in this case it contains a color picker but in the 2d spectrum, there are more options so what I have to do?

**Comment by lpatiny** [comment]
For 2D I would suggest a square cut in diagonal with the 2 colors.
